samples/outbox/store-and-forward/Version_5/Sender/app.config

```diff
@@ -1,17 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@SimonCropp that makes sense. If V4 of SQL transport targets V6 it is likely to require a separate code in this sample so I'll go with transport versions.